src/modules/twap/containers/TwapFormWidget/tooltips.tsx

```diff
@@ -0,0 +1,56 @@
+export interface LabelTooltipObject {
+  label: string
+  tooltip?: JSX.Element
+}
+
+export interface LabelTooltipItems {
+  [key: string]: LabelTooltipObject
+}
+
+export const LABELS_TOOLTIPS: LabelTooltipItems = {
```

Comment:
I'm not sure why is better to export a big const with all tooltips instead of individual tooltips:

i.e. `export numberOfPartsTooltip: TooltipInfo  = { .... }`

then its easier to tree-shake, and check the call hierarchy, and is less verbose when you use it in the other file

Reply:
Since it's used only in one place, I think it's ok